content/ai/concepts/minimax-algorithm/ai-minimax.md

```diff
@@ -0,0 +1,137 @@
+---
+Title: 'Minimax algorithm in AI'
+Description: 'Minimax algortihm is a recursive algorithm used in decision making and game theory.'
+Subjects:
+  - 'AI'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Algorithms'
+  - 'Graphs'
+  - 'Graph Search'
+CatalogContent:
+  - 'paths/data-science'
+---
+
+The **minimax algorithm** is a fundamental concept in artificial intelligence used for decision making in games and other scenarios.
+It is widely used in two player games where players alternate turns, such as tic-tac-toe, chess, backgammon, etc. 
+In these games, each player wants to make the best or most optimal move to maximize their chances of winning, while also trying to minimize their opponent's chances. Therefore, the name of the algorithm is a combination of two words: minimize and maximize.
+The minimax algorithm helps the computer or AI decide its next move by considering all possible moves it can make and all possible responses from the opponent. It does this by simulating the game's future moves and outcomes.
```

Comment:
```suggestion
The **minimax algorithm** is a fundamental concept in artificial intelligence used for decision making in games and other scenarios.
It is widely used in two-player games where players alternate turns, such as tic-tac-toe, chess, backgammon, etc. 
 In these games, each player wants to make the best or most optimal move to maximize their chances of winning, while also trying to minimize their opponent's chances. Therefore, the algorithm's name is a combination of two words: minimize and maximize. The minimax algorithm helps the computer or AI decide its next move by considering all possible moves it can make and all possible responses from the opponent. It does this by simulating the game's future moves and outcomes.
```